lib/framework/i18n.cpp

```diff
@@ -168,6 +170,7 @@ static const struct
 {
 	{ "",   N_("System locale"), "", "" },
 #  if defined(ENABLE_NLS)
+	{ "ar_SA", LANG_NAME_ARABIC, "ar_SA.UTF-8", "ar_SA"},
```

Comment:
@mt3d I'm wondering if the fallback here should be 
```suggestion
	{ "ar_SA", LANG_NAME_ARABIC, "ar_SA.UTF-8", "ar"},
```
instead?

Reply:
I missed that! Thanks.